src/components/TextField/components/Resizer/Resizer.tsx

```diff
@@ -56,29 +57,28 @@ export default class Resizer extends React.PureComponent<Props, never> {
   }
 
   private handleHeightCheck = () => {
-    if (this.contentNode == null || this.minimumLinesNode == null) {
-      return;
+    if (this.animationFrame) {
+      cancelAnimationFrame(this.animationFrame);
     }
 
-    const contentHeight = this.contentNode.offsetHeight;
-    const minimumHeight = this.setMinimumLinesNode
```

Comment:
This condition is always true because `this.setMinimumLinesNode` is a function and `!!Function` is truthy.